packages/host/src/Admin/SyncSettings.tsx

```diff
@@ -0,0 +1,194 @@
+import React, { FC, useState } from 'react';
+import { useTranslation } from '@common/intl';
+import {
+  BasicTextInput,
+  CircularProgress,
+  Grid,
+  LoadingButton,
+  NumericTextInput,
+  SaveIcon,
+  Typography,
+  UpdateSyncSettingsInput,
+  useNotification,
+} from '@openmsupply-client/common';
+import { useHost } from '../api/hooks';
+import { Setting } from './Setting';
+
+type SyncSettings = Omit<UpdateSyncSettingsInput, '__typename'>;
+
+const isValid = (syncSettings: SyncSettings | null) => {
+  if (!syncSettings) return false;
+  return (
+    !!syncSettings.url &&
+    !!syncSettings.username &&
```

Comment:
The api actually checks if the url is a valid url and if the username is not empty... Otherwise I don't do any validation

Reply:
that's right.. I remember seeing that in the PR and was going to make use of it here. then forgot. Should perhaps add URL validation here.. but I rather like the server validating and having that in one place. I've just managed to save an invalid URL and created chaos with the server falling back to bootstrap mode! 🤣

Reply:
:) it just checks if the URL is well formatted, I don't try to connect to it...